### PR TITLE
Xz/cli only

### PR DESCRIPTION
**Pull request by zhangxu0307**
support CLI only mode

### CODE REVIEW

File: website/docs/session.md [hunk 1]
```diff
@@ -61,4 +62,30 @@ The execution of the generated python code above has succeeded
 The result of above Python code after execution is:
 array([0.09918602, 0.68732778, 0.44413814, 0.4756623 , 0.48302334,
        0.8286594 , 0.80994359, 0.35677263, 0.45719317, 0.68240194])
-`````
\ No newline at end of file
+``````
+- plugin_only_mode: please refer to [plugin_only_mode](./customization/plugin/plugin_only.md) for more details.
+- cli_only_mode: allow users to directly communicate with the Command Line Interface (CLI) in natural language.
+  CodeInterpreter will generate CLI commands (e.g., bash/powershell), instead of Python code, to fullfill the user's request.
```
Comment (liqul) [inline website/docs/session.md:68]: fulfill?

File: taskweaver/session/session.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 from taskweaver.code_interpreter import CodeInterpreter, CodeInterpreterPluginOnly
 from taskweaver.code_interpreter.code_executor import CodeExecutor
+from taskweaver.code_interpreter.code_interpreter_cli_only import CodeInterpreterCLIOnly
```
Comment (liqul) [inline taskweaver/session/session.py:9]: add to __init__.py as others?

File: taskweaver/session/session.py [hunk 3]
```diff
@@ -21,6 +22,7 @@ def _configure(self) -> None:
         self.code_interpreter_only = self._get_bool("code_interpreter_only", False)
         self.max_internal_chat_round_num = self._get_int("max_internal_chat_round_num", 10)
         self.plugin_only_mode = self._get_bool("plugin_only_mode", False)
+        self.cli_only_mode = self._get_bool("cli_only_mode", False)
```
Comment (liqul) [inline taskweaver/session/session.py:25]: change to only one config mode?

File: taskweaver/code_interpreter/code_generator/code_generator_prompt_cli_only.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+version: 0.1
+content: |-
+    {ROLE_NAME} can understand the user request and generate command line interface (CLI) commands to complete tasks.
+    {ROLE_NAME} need to follow the below instructions to generate the CLI commands:
+      - Understand the user request and identify the task to be performed.
+      - Generate the CLI command under OS:{OS_NAME} environment. 
+        - If the os is Windows, use the `powershell` command prompt. {ROLE_NAME} need to write as `powershell -Command "Your PowerShell command here"`  .
+        - If the os is Linux, use the `bash` command prompt.
```
Comment (liqul) [inline taskweaver/code_interpreter/code_generator/code_generator_prompt_cli_only.yaml:8]: add macos?

File: taskweaver/session/session.py (hunk 2 above)
Comment (zhangxu0307) [reply taskweaver/session/session.py]: moved

File: website/docs/session.md (hunk 1 above)
Comment (zhangxu0307) [reply website/docs/session.md]: revised

File: taskweaver/session/session.py (hunk 3 above)
Comment (zhangxu0307) [reply taskweaver/session/session.py]: merged as one config parameter `session.code_exec_mode`

File: taskweaver/code_interpreter/code_generator/code_generator_prompt_cli_only.yaml (hunk 4 above)
Comment (zhangxu0307) [reply taskweaver/code_interpreter/code_generator/code_generator_prompt_cli_only.yaml:8]: added

**Review by liqul — APPROVED** [review]

**Pull request merged**